src/UI/class-settings-page.php

```diff
@@ -598,6 +612,30 @@ public function print_multiple_checkboxes( array $args ): void {
 		$this->print_description_text( $args );
 	}
 
+	/**
+	 * Print out a "single-image browse control" which includes a text
+	 * input to store image path and a button to browse for images.
+	 *
+	 * @param array $args The arguments for the control.
+	 * @return void
+	 */
+	public function print_media_single_image( array $args ) {
+		$key         = esc_attr( $args['option_key'] );
+		$input_value = $this->parsely->get_options()[ $key ];
+		$input_name  = Parsely::OPTIONS_KEY . "[$key]";
+		$button_text = __( 'Browse', 'wp-parsely' );
+		?>
+
+		<fieldset class="media-single-image" id="media-single-image-<?php echo esc_attr( $key ); ?>">
+			<legend class="screen-reader-text"><span><?php echo esc_html( $args['title'] ); ?></span></legend>
+			<input class="large-text file-path" type="text" name="<?php echo esc_attr( $input_name ); ?>" value="<?php echo esc_attr( $input_value ); ?>" />
+			<button data-option="<?php echo esc_attr( $key ); ?>" class="browse button" style="margin-top: 0.4rem" type="button"><?php echo esc_html( $button_text ); ?></button>
```

Comment:
Does the custom style still work zoomed into or out of the page (CMD-+)?

Reply:
Here's what I get in Firefox and Chromium. I don't see inconsistencies with the other elements as I zoom in/out. So my response to this question would be yes.

![image](https://user-images.githubusercontent.com/23142906/149489140-ce9b9f0e-a2bd-43bf-97bb-e8955e43443c.png)

![image](https://user-images.githubusercontent.com/23142906/149489602-4a197a2b-2d16-4330-b876-2a13b6fac0fb.png)

![image](https://user-images.githubusercontent.com/23142906/149489694-a5e9cd8e-1ca0-40fd-8e4a-0f995a0c69be.png)

Reply:
Custom style is not relevant anymore so I'm marking this as resolved.